docs/tutorials/buttons.rst

```diff
@@ -70,14 +70,9 @@ niezwykle szybko.
 Pętle zdarzeń
 +++++++++++
 
-Często chcesz aby program poczekał na jakieś zdarzenie. Żeby to uzyskać, musisz
-zamknąć w pętli kawałek kodu definiujący jak reagować na oczekiwane zdarzenia,
-takie jak naciśnięcie przycisku.
-
-Aby uzyskać
-to musisz zamknąć odpowiedni kawałek kodu w pętli, który będzie miał
-zdefiniowane w jaki sposób zareagować na oczekiwane zdarzenie takie jak
-naciśnięcie przycisku.
+Często chcesz aby program poczekał na jakieś zdarzenie. Aby uzyskać to musisz
+zamknąć odpowiedni kawałek kodu w pętli, który będzie miał zdefiniowane w jaki
+sposób zareagować na oczekiwane zdarzenie takie jak naciśnięcie przycisku.
```

Comment:
Poprzednia wersja była lepsza, moim zdaniem.

Po pierwsze "aby uzyskać to" chyba nie jest za bardzo po polsku, jest kalką słowo w słowo z szyku zdania po angielsku i brzmi dziwnie. Po drugie, najpierw mówimy o kodzie, potem o pętli, i potem nagle znowu o kodzie — trudno się to czyta, mimo, że mają różne rodzaje. Zamienienie miejscami pętli i kodu powoduje, że nie przeskakujemy tak.